packages/transit-vehicle-overlay/package.json

```diff
@@ -10,15 +10,15 @@
   "private": false,
   "dependencies": {
     "@opentripplanner/base-map": "^3.1.0",
-    "@opentripplanner/core-utils": "^11.4.0",
+    "@opentripplanner/core-utils": "^11.4.1",
```

Comment:
Note that this is to accommodate tests only, and that no release will be triggered for this package.